sympy/core/function.py

```diff
@@ -1291,7 +1299,7 @@ def __new__(cls, expr, *variables, **kwargs):
         if len(variable_count) == 0:
             return expr
 
-        evaluate = kwargs.get('evaluate', False)
+        evaluate = fuzzy_bool(kwargs.get('evaluate', None))
```

Comment:
I don't see what fuzzy_bool is doing here that is useful

Reply:
It guarantees that the evaluate value is now True, False or None just as `bool(foo)` would guarantee that you are working only with True and False. After such recasting you can confidently test like `foo is True`.

Reply:
Is there a reason to allow the user to pass in anything other than True, False or None?